tests/iso/test_get_country_code.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from returns.primitives.exceptions import UnwrapFailedError
+
+from kaiba.iso import get_country_code
+
+# Norway, NO, NOR, 578
```

Comment:
@thomasborgen There is no real need in this comment, maybe only for development convenience, but I bet it can be removed at this stage.